src-tauri/tauri.conf.json

```diff
@@ -44,7 +44,12 @@
       }
     },
     "updater": {
-      "active": false
+      "active": true,
+      "endpoints": [
+        "https://tauri-update-server.serge988.workers.dev/v1/{{target}}/{{arch}}/{{current_version}}"
```

Comment:
Should this URL be updated from `serge988` to something else?

Reply:
Do we need different domain for this? As I understand its for demo purposes only

Reply:
@nazar-pc I will use another subdomain